db/schema.rb

```diff
@@ -126,6 +126,8 @@
     t.text "terms_and_conditions"
     t.integer "board_layout", default: 0, null: false
     t.boolean "registration_enabled", default: true, null: false
+    t.boolean "request_team_location", default: false, null: false
+    t.boolean "location_required", default: false, null: false
```

Comment:
Are these fields displayed in rails_admin?

Reply:
Yes, however fields will display on top/bottom, I'm going to have to define a fields list for all fields to make them not show up in a weird spot.